packages/react/src/components/LightboxMediaViewer/__stories__/LightboxMediaViewer.stories.js

```diff
@@ -89,6 +89,16 @@ EmbeddedVideoPlayer.story = {
         media: {
           src: '1_9h94wo6b',
           type: 'video',
+          title: text(
+            'title (required)',
+            'Curabitur malesuada varius mi eu posuere',
+            groupId
+          ),
+          description: text(
+            'description (required)',
```

Comment:
Great work @rsebade ! One thing, the `title` and `description` I believe are `optional` and not `required`, since if this isn't defined it will just pull from the Kaltura API data.

Reply:
@jeffchew Good catch! Just updated this.